wl/server/SyncThread.java

```diff
@@ -65,14 +65,14 @@ public void run() {
 				Thread.sleep(then - now);
 
 				Utils.log("Waking up for " + (phase == 0 ? "full" : "SQL-only") + " sync.");
-				for (Utils.Databases db : Utils.Databases.values()) Utils.sql(db, "show tables");
+				for (Utils.Databases db : Utils.Databases.values()) Utils.sql(Utils.config(db.db_name), "show tables");
```

Comment:
Why would this be necessary, shouldn't this be reverted?

Reply:
Is this for-loop needed at all? As far i can see this get not logged or has a meaning at all.

Reply:
See comment 20 lines above ;)

If the server is inactive for eight hours MySQL kills the database connections, and reopening them when again needed is near-impossible. So we run some cheap query (the result of which we ignore) every six hours on every database to prevent that.